projects/bignum-fuzzer/Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN apt-get update && apt-get install -y \
 
 # Install Python packages from PyPI
 RUN pip3 install jinja2
+RUN pip3 install jsonschema
```

Comment:
We haven't been mentioning the new Python requirements in changelog entries because they aren't needed if you grab a release of Mbed TLS, only if you grab the development branch (the file comes pre-generated in releases). Would it help if we did?